cpp/oneapi/dal/algo/finiteness_checker/backend/cpu/compute_kernel.hpp

```diff
@@ -24,9 +24,9 @@ namespace oneapi::dal::finiteness_checker::backend {
 
 template <typename Float, typename Method, typename Task>
 struct compute_kernel_cpu {
-    bool operator()(const dal::backend::context_cpu& ctx,
-                    const detail::descriptor_base<Task>& params,
-                    const compute_input<Task>& input) const;
+    compute_result<Task> operator()(const dal::backend::context_cpu& ctx,
+                                    const detail::descriptor_base<Task>& params,
+                                    const compute_input<Task>& input) const;
 
 #ifdef ONEDAL_DATA_PARALLEL
```

Comment:
Do we want to have it, or it can be deleted?

Reply:
it was about
    void operator()(const dal::backend::context_cpu& ctx,
                    const detail::descriptor_base<Task>& params,
                    const table& data,
                    bool& res) const;